### PR TITLE
UCP: Migrate `dataForTiDBClusterInfo` from package `infoschema` to `executor`

### PR DESCRIPTION
**Pull request by yzwqf**
### What problem does this PR solve? 
UCP [#15031](https://github.com/pingcap/tidb/issues/15031)

### What is changed and how it works?
Migrate dataForTiDBClusterInfo from package infoschema to executor.

### Check List 

Tests 

 - Unit test
 - Integration test

### CODE REVIEW

**Comment by zz-jason** [comment]
Hi @yzwqf, thanks for your contribution! The failed UT seems is unstable, which was a known issue: https://github.com/pingcap/tidb/issues/15091. It's under investigation, we'll fix it ASAP.

**Comment by yzwqf** [comment]
> Hi @yzwqf, thanks for your contribution! The failed UT seems is unstable, which is a known issue: #15091. It's under investigation, we'll fix it ASAP.
> 
> Also, please sign the CLA so we can get this PR merged.

thx, i also try to figure it out

File: executor/infoschema_reader_test.go [hunk 1]
```diff
@@ -309,3 +326,207 @@ func (s *testInfoschemaTableSuite) TestPartitionsTable(c *C) {
 
 	tk.MustExec("DROP TABLE `test_partitions`;")
 }
+
+var _ = SerialSuites(&testInfoschemaClusterTableSuite{testInfoschemaTableSuite: &testInfoschemaTableSuite{}})
```
Comment (lonng) [inline executor/infoschema_reader_test.go]: Do we realy need to nest the `testInfoschemaTableSuite` in the `testInfoschemaClusterTableSuite `?

Comment (yzwqf) [reply executor/infoschema_reader_test.go]: unneccessaryly, just for reuse code like done in the tables_test.go before, it is ok to decouple.

**Comment by zz-jason** [comment]
@yzwqf gofmt failed, please run `make check` in your machine to fix it.

**Comment by yzwqf** [comment]
> @yzwqf gofmt failed, please run `make check` in your machine to fix it.

I'm new to git and very depressed, how is it now...

**Comment by lonng** [comment]
Hi, @yzwqf Seems you do some misoperation on git, maybe you can do the following instruction to pull it back on the right path.
```
git checkout myfeature
git reset --hard a000858
make fmt
git commit -am 
git push -f
```

**Comment by zz-jason** [comment]
@yzwqf please resolve the conflicts.

**Comment by reafans** [comment]
Rest LGTM, Hi @yzwqf, thanks for your contribution. The failed check_dev shows that the code need format, just use `make fmt` for your changed code and commit them to fix it.

File: executor/infoschema_reader_test.go [hunk 2]
```diff
@@ -14,18 +14,34 @@
 package executor_test
 
 import (
+	"crypto/tls"
+	"fmt"
 	"github.com/pingcap/tidb/statistics"
 	"github.com/pingcap/tidb/util/testutil"
+	"net"
```
Comment (crazycs520) [inline executor/infoschema_reader_test.go]: move line #21 ~ #25 after line #18

Comment (yzwqf) [reply executor/infoschema_reader_test.go]: 'make fmt' error...

Comment (yzwqf) [reply executor/infoschema_reader_test.go]: cannot commit

**Comment by reafans** [comment]
lgtm

**Pull request merged**